infra/lib/charts/prow-secrets.ts

```diff
@@ -4,46 +4,50 @@ import * as kplus from 'cdk8s-plus-20';
 import { PROW_NAMESPACE, PROW_JOB_NAMESPACE } from '../test-ci-stack';
 
 export interface ProwSecretsChartProps {
-  readonly botPersonalAccessToken: string;
-  readonly webhookHMACToken: string;
+  readonly githubAppId: string;
+  readonly githubAppPrivateKey: string;
+  readonly githubAppWebhookSecret: string;
 }
 
 export class ProwSecretsChart extends cdk8s.Chart {
-  readonly botPATSecret: kplus.Secret;
-  // github token to be used by prowjobs in PROW_JOB_NAMESPACE
-  readonly prowjobBotPATSecret: kplus.Secret;
-  readonly webhookHMACSecret: kplus.Secret;
+  readonly githubToken: kplus.Secret;
+  // github client secret to be used by prowjobs in PROW_JOB_NAMESPACE
+  readonly prowjobGithubToken: kplus.Secret;
+  readonly hmacToken: kplus.Secret;
 
   constructor(scope: constructs.Construct, id: string, props: ProwSecretsChartProps) {
     super(scope, id);
 
-    if (props.botPersonalAccessToken === undefined || props.webhookHMACToken === undefined) {
-      throw new Error(`Expected bot personal access token and webhook HMAC token to be specified`);
+    if (props.githubAppPrivateKey === undefined || props.githubAppWebhookSecret === undefined || props.githubAppId === undefined) {
+      console.trace()
+      throw new Error(`Expected GitHub app ID & app private key & app webhook HMAC token to be specified`);
     }
 
-    this.botPATSecret = new kplus.Secret(this, 'github-token', {
+    this.githubToken = new kplus.Secret(this, 'github-token', {
       stringData: {
-        'token': props.botPersonalAccessToken
+        'cert': props.githubAppPrivateKey,
+        'appid': props.githubAppId
```

Comment:
nit: Follow same casing rules as earlier (`appId`)

Reply:
Won't do -- the [deployment Getting Started guide](https://github.com/kubernetes-sigs/prow/blob/main/site/content/en/docs/Legacy%20Snapshot/prow/getting_started_deploy.md#create-the-github-secrets) uses the `appid` spelling, so I think it's best to stay consistent.

Reply:
I am fine with it being `appid` in the Secret, to stay consistent with their secret, but we should at least stick to our casing for the `values.yaml` file.

Reply:
OK -- I think it's actually more confusing this way, but I'll defer to your recommendation here.

Done